Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/View.purs

```diff
@@ -2350,6 +2536,11 @@ quoteListModelView push state =
 emptyTextView :: forall w. HomeScreenState ->  PrestoDOM (Effect Unit) w
 emptyTextView state = textView [text "", width $ if os == "IOS" then V 1 else V 0]
 
+
+------------------------ emptyTextView ---------------------------
+emptyTextViewNoHeight :: forall w. HomeScreenState ->  PrestoDOM (Effect Unit) w
```

Comment:
can't we use emptyTextView itself for this?

Reply:
No, as it was taking some default height. But I didn't want the extra space to be there